google/cloud/pubsub/samples/samples.cc

```diff
@@ -839,7 +839,6 @@ void ExampleStatusOr(google::cloud::pubsub::TopicAdminClient client,
 
 void CreateAvroSchema(google::cloud::pubsub::SchemaServiceClient client,
                       std::vector<std::string> const& argv) {
-  //! [START pubsub_create_avro_schema] [create-avro-schema]
```

Comment:
https://github.com/googleapis/google-cloud-cpp/pull/13527#discussion_r1472981689

AFAIK, our documentation tooling has not changed in the last 48 hours and we still need to follow this process.

Reply:
Reverted the changes. I'll delete them in a later PR.